src/renderer/App.vue

```diff
@@ -87,6 +89,19 @@ html, body, #app {
   button {
     font-family: var(--specified-fonts);
   }
+
+  ::-webkit-scrollbar {
+    width: 8px;
+  }
+
+  ::-webkit-scrollbar-track {
```

Comment:
__[stylelint]__ Expected selector "html ::-webkit-scrollbar-track" to come before selector "#app ::-webkit-scrollbar" (no-descending-specificity) ([link](https://sider.review/gh/repos/124234495/pulls/889#issue-3995952))

<sub>posted by Sider</sub>